Integrations/integration-VxStream.yml

```diff
@@ -428,3 +459,19 @@ script:
       description: 'environment ID to submit file to(get all via vx-get-environments) '
     description: Retrieve result data upon a file. NOTE - This command returns a file
       (also - minimum required authorization is `default`)
+  - name: vx-detonate-file
+    arguments:
+    - name: entryId
+      required: true
+      description: War-room entry ID of sample file
+    - name: environmentID
+      required: true
```

Comment:
@michalgold we consider to make `environmentID` optional - and if its not provided - it will choose the first environment ID that it finds. What do you think?

Reply:
I think that we could make it optional and when it's not provided use environment ID 100 as default. 
![image](https://user-images.githubusercontent.com/30618865/33150240-42ef43a0-cfdb-11e7-8fce-481868f12fed.png)
If they change this/environment ID 100 is not found, let's use the first windows environment ID we can find.